n8n/helm/n8n/Chart.yaml

```diff
@@ -3,8 +3,8 @@ name: n8n
 description: helm chart for n8n
 type: application
 version: 0.1.6
```

Comment:
Can you bump the chart version? This is required to push a new helm chart to Plural.

Reply:
done